docs/standard/containerized-lifecycle-architecture/design-develop-containerized-apps/state-and-data-in-docker-applications.md

```diff
@@ -39,7 +39,7 @@ Due to the inability to manage data shared between containers that run on separa
 
 Therefore, regular data volumes are a good mechanism to work with trace files, temporal files, or any similar concept that won't affect the business data consistency if or when your containers are moved across multiple hosts.
 
-Volume plug-ins like [Flocker](https://clusterhq.com/flocker/) provide data across all hosts in a cluster. Although not all volume plug-ins are created equally, volume plug-ins typically provide externalized persistent reliable storage from the immutable containers.
+Volume plug-ins like *Flocker* provide data across all hosts in a cluster. Although not all volume plug-ins are created equally, volume plug-ins typically provide externalized persistent reliable storage from the immutable containers.
```

Comment:
why not link to https://github.com/ClusterHQ/flocker?

Reply:
Because from what I understand, there's no active development since ClusterHQ went out of business.

Reply:
Maybe we should just remove the reference completely.  @CESARDELATORRE, what do you think?